Source/Requests/HARequestType.swift

```diff
@@ -1,9 +1,13 @@
+import Foundation
+
 /// The command to issue
 public enum HARequestType: Hashable, Comparable, ExpressibleByStringLiteral {
     /// Sent over WebSocket, the command of the request
     case webSocket(String)
     /// Sent over REST, the HTTP method to use and the post-`api/` path
     case rest(HAHTTPMethod, String)
+    /// Sent over WebSocket, the stt binary handler id
+    case sttData(UInt8)
```

Comment:
Can you make a struct out of the UInt8 here whose value is the integer?

Reply:
Besides HAHTTPMethod the other request types we are using raw values such as String, why a struct just for this case?

Reply:
Hmm. I think the biggest reason is readability. I can see `.webSocket("do_thing")` and have an idea what it means, but I can't see `.sttData(2)` and know what that is unless I've memorized the table of values for 2.